ansible/k8s-k8sproxy.yml

```diff
@@ -0,0 +1,61 @@
+# Setup the K8s shared web proxy
+
+# Load hostvars
+- hosts: >
+    {{ idr_environment | default('k8s') }}-k8svae-hosts
```

Comment:
I'd be for a round of naming discussions around these. e.g. I assume that `k8svae` actually equals `vae48`

Reply:
No, in line with the naming of the reset of the IDR it would be the idr-environment that changes. Currently `idr-environment = k8s01`. I don't think trying to use the core IDR versions for the decoupled VAE makes sense.